python/GafferCortexUI/StringParameterValueWidget.py

```diff
@@ -55,7 +57,20 @@ def __init__( self, parameterHandler, **kw ) :
 			multiLine = parameterHandler.parameter().userData()["UI"]["multiLine"].value
 
 		if multiLine :
+			
+			# by default the multi-line widget gets auto-expanded in the op interface which works nicely when
```

Comment:
cool!!! I'm still finding my way to use the plugs metadata, thanks for pointing me that